src/root/schemas/field-report-form.js

```diff
@@ -27,10 +27,10 @@ export const step1 = {
     districts: {
       type: 'array'
     },
-    disasterType: {
-      type: 'string',
-      enum: getValidValues(formData.disasterType, 'value')
-    },
+    // disasterType: {
```

Comment:
@frozenhelium @batpad Similar to `utils/utils.js`, I couldn't find a way to use the `disasterTypesSelect` array from the `state` here.

Reply:
This is non-ideal, but okay, there should be no reasonable way for the user to get an invalid value here, and we are validating on the backend.

I would like to find a good pattern to get the schema validator to use values from the state, but this is probably a bit involved. If we figure that pattern out, we can implement it for countries as well, but I wouldn't worry too much about this, this seems fine.

Reply:
I'll ticket this out for later.